configure.ac

```diff
@@ -37,7 +37,7 @@ programs_man_section=1
 libraries_man_section=3
 
 # Command to build executalbes
-mkexe="\$(CC) \$(OC_LDFLAGS)"
+mkexe="\$(CC) \$(OC_CFLAGS) \$(CFLAGS) \$(OC_LDFLAGS)"
```

Comment:
I think it's worth a comment here that `$(OC_CFLAGS)` and `$(CFLAGS)` don't get included on Windows because flexlink is called, rather than the C compiler?